zerver/lib/queue.py

```diff
@@ -11,14 +13,15 @@
 from collections import defaultdict
 
 from zerver.lib.utils import statsd
-from typing import Any, Callable
+from typing import Any, Callable, Optional
 
 # This simple queuing library doesn't expose much of the power of
 # rabbitmq/pika's queuing system; its purpose is to just provide an
 # interface for external files to put things into queues and take them
 # out from bots without having to import pika code all over our codebase.
 class SimpleQueueClient(object):
     def __init__(self):
+        # type: () -> None
         self.log = logging.getLogger('zulip.queue')
         self.queues = set() # type: Set[str]
         self.channel = None # type: Any
```

Comment:
I think this should be typed as `Optional[BlockingChannel]`, since line 37 (in `_connect`) is
```py
        self.channel    = self.connection.channel() # type: BlockingChannel
```
and line 40 (in `_reconnect`) is
```py
self.channel = None
```